spec/forms/journeys/teacher_student_loan_reimbursement/still_teaching_form_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "rails_helper"
+
+RSpec.describe Journeys::TeacherStudentLoanReimbursement::StillTeachingForm, type: :model do
+  before { create(:journey_configuration, :student_loans) }
+
+  let(:claim_school) { build(:school, :student_loans_eligible) }
+  let(:eligibility) { create(:student_loans_eligibility, claim_school:, employment_status: nil) }
+  let(:claim) { create(:claim, policy: Policies::StudentLoans, eligibility:) }
+  let(:current_claim) { CurrentClaim.new(claims: [claim]) }
+  let(:claim_params) { {} }
+
+  subject(:form) do
+    described_class.new(
+      journey: Journeys::TeacherStudentLoanReimbursement,
+      claim: current_claim,
+      params: ActionController::Parameters.new(claim: claim_params)
+    )
+  end
+
+  describe "validations" do
+    it do
+      should validate_presence_of(:employment_status).with_message(
+        "Select if you still work at #{claim_school.name}, another school or no longer teach in England"
+      )
+    end
+  end
+
+  describe "#error_message" do
+    context "when the school is closed" do
+      let(:claim_school) { build(:school, :student_loans_eligible, close_date: Date.yesterday) }
+
+      it "does not contain the school name" do
+        expect(form.error_message).to eq("Select yes if you are still employed to teach at a school in England")
```

Comment:
This is also a bug on master. If the school isn't open, we only show two options to the question "Are you still employed to teach at a school in England?", yes or no. We shouldn't be showing the closed school name in the error message if it's not an option to pick.

On master:
<img width="705" alt="Screenshot 2024-05-08 at 10 47 40" src="https://github.com/DFE-Digital/claim-additional-payments-for-teaching/assets/18436946/910e0d83-56ff-4e14-9d5b-770dcc68f72b">

Reply:
Now says "Select if you are still employed to teach at a school in England"